website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -180,6 +180,26 @@ default value in the `"/sys/config/ui"` [API endpoint](/vault/api-docs/system/co
   connecting client's IP, for example `3.4.5.6`. Note this requires the load balancer 
   to send the connecting client's IP in the `X-Forwarded-For` header.
 
+- `x_forwarded_for_client_cert_header` `(string: "")` –
```

Comment:
I would use x_forwarded_for_tls_client_cert_header.

Reply:
That name was picked to match the naming of the rest of the variables: `tls_require_and_verify_client_cert` and `tls_disable_client_certs`. For both of those variables, it is just `client_cert`.